awx_collection/plugins/module_utils/tower_api.py

```diff
@@ -588,3 +589,42 @@ def is_job_done(self, job_status):
             return False
         else:
             return True
+
+    def wait_on_url(self, url, object_name, object_type, timeout=30, interval=10):
+        # Grab our start time to compare against for the timeout
+        start = time.time()
+        result = self.get_endpoint(url)
+        while not result['json']['finished']:
+            # If we are past our time out fail with a message
+            if timeout and timeout < time.time() - start:
+                # Account for Legacy messages
+                if object_type is 'legacy_job_wait':
+                    self.json_output['msg'] = 'Monitoring of Job - {1} aborted due to timeout'.format(object_name)
```

Comment:
This format statement uses `{1}` but it looks like it should use the 0 index.

Reply:
Updated to 0